content/docs/testing-recipes.md

```diff
@@ -56,6 +56,8 @@ You may use a different pattern, but keep in mind that we want to execute the cl
 
 ### `act()` {#act}
 
+> Please refer to the [react-act-examples/sync.md](https://github.com/threepointone/react-act-examples/blob/master/sync.md) for details on how act () works.
```

Comment:
@threepointone this still the best resource?

Reply:
I'll work on something better.